gobblin-dataset-config/src/test/resources/filebasedConfig/v1.0.1/application.conf

```diff
@@ -0,0 +1,40 @@
+// circular dependency for config-tag.t1.t2.t3 -> config-tag.tag1.tag2 -> config-tag.t1.t2.t3
```

Comment:
will remote those files after I made all the unit testing working for the new code as I may need to for test cases